### ISSUE TITLE
[Re-occuring] Endpoint returned in not consistent with AWS (Lambda)

### ISSUE THREAD

**Issue by OnePunMan** [localstack/serverless-localstack#126]
Hi, this is a reoccurring issue - https://github.com/localstack/serverless-localstack/issues/28 and supposedly already solved but I'm noticing it again.

I made a comment on the old issue a few days ago (https://github.com/localstack/serverless-localstack/issues/28#issuecomment-750943803) but there hasn't been any responses so I'll open a new one here. To reiterate what I said:

Hi, I am still encountering this on version 0.12.3 when I deploy on localstack Lambda. upon deploying using sls, I get this:
```
Service Information
service: my-service
stage: local
region: us-east-1
stack: my-service-local
resources: 11
api keys:
  None
endpoints:
  http://localhost:4566/restapis/0i514w1of2/local/_user_request_
functions:
  hello: my-service-local-hello
layers:
  None
```

I had to invoke the endpoint the old fashion way by following https://github.com/localstack/serverless-localstack/issues/28#issuecomment-500199803 (i.e. `http://localhost:4566/restapis/0i514w1of2/local/_user_request_/hello`). You can find the source code I used to reproduce the issue [here](https://github.com/OnePunMan/serverless-localstack-templates/tree/master/templates/python-lambda). It is a very simple template handler.

**Comment by whummer** [comment]
Hi @OnePunMan , thanks for reporting. Not sure which exact issue you're facing, but as far as I can tell that this is the expected output.

Please refer to this comment to get more details on which patterns we're using for the local invocation endpoints in LocalStack: https://github.com/localstack/serverless-localstack/issues/28#issuecomment-500199803

Can you please share some more details about the actual issue you're facing.  Which output would you have expected, what do you think needs to be fixed? Thanks

**Comment by OnePunMan** [comment]
Hi, thanks for the response! The issue was rather, the inconsistency between the output format on localstack compared to real AWS. On localstack, the endpoint is composed by concatenating the function name and the base URL. On AWS it returns the URL within preformatted.

The original question in the old issue was exactly this, and from the comment https://github.com/localstack/serverless-localstack/issues/28#issuecomment-523196519, it was supposedly fixed it as shown in the code snippet (the function name was appended to the base URL). 
```
endpoints:
  POST - http://localhost:4567/restapis/txwn2s2eog/local/_user_request_/v1/request
functions:
  http_request: test-local-http_request
```


I reported this as reoccurring because I encountered the same issue again (on localstack lambda), the output URL was `http://localhost:4566/restapis/0i514w1of2/local/_user_request_`, but I was expecting `http://localhost:4566/restapis/0i514w1of2/local/_user_request_/hello` since the comment mentioned that it was fixed.

Hope this helps clarify some things, thank you!
